crp_APL/controllers/ctrl_vehicle_control_lat_compensatory/include/ctrl_vehicle_control_lat_compensatory/ctrl_vehicle_control_lat_compensatory.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef CRP_APL_CTRL_VEHICLE_LAT_COMPENSATORY_HPP
+#define CRP_APL_CTRL_VEHICLE_LAT_COMPENSATORY_HPP
+
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include <math.h>
+#include "rclcpp/rclcpp.hpp"
+#include "geometry_msgs/msg/pose.hpp"
+#include "std_msgs/msg/float32_multi_array.hpp"
+#include "std_msgs/msg/float32.hpp"
+#include "crp_msgs/msg/ego.hpp"
+
+#include "autoware_control_msgs/msg/control.hpp"
+#include "autoware_planning_msgs/msg/trajectory.hpp"
+#include "autoware_vehicle_msgs/msg/steering_report.hpp"
+
+#include "ctrl_vehicle_control/controller_inputs.hpp"
+#include "ctrl_vehicle_control/controller_outputs.hpp"
+#include "lib/utils.hpp"
+#include "compensatory_model/compensatory_model.hpp"
+
+namespace crp
+{
+    namespace apl
+    {
+        class CtrlVehicleControlLat : public rclcpp::Node
+        {
+            public:
+                CtrlVehicleControlLat();
+
+            private:
+                // VARIABLES
+                crp::apl::ControlInput input;
+                crp::apl::ControlOutput output;
+                crp::apl::ControlParams params;
```

Comment:
please ezeket is írd már át m_ -el kezdődőre... bocsánat, csak hogy legyen egységes